LICENSE

```diff
@@ -0,0 +1,22 @@
+
+The MIT License (MIT)
+
+Copyright (c) UTN-LIS
```

Comment:
Note that I didn't specify year ranges, we could probably follow this approach to avoid being updated the dates when we made modifications to the file, or use this approach:
```
Copyright (c) 2013 - present UTN-LIS
```